static/app/components/menuItem.tsx

```diff
@@ -168,14 +170,14 @@ const MenuItem = ({
   );
 };
 
-type MenuListItemProps = {
+interface MenuListItemProps extends React.LiHTMLAttributes<HTMLLIElement> {
```

Comment:
Wait what's the difference between this and `HTMLAttributes` 🤔

Reply:
iirc this is just the react preset